src/HDF5.jl

```diff
@@ -19,7 +19,8 @@ copy_object, open_object, delete_object,
 create_datatype, commit_datatype, open_datatype,
 create_property,
 group_info, object_info,
-dataspace, datatype
+dataspace, datatype,
+Filters, Drivers
```

Comment:
Did you really intend to export these types? Currently the types in HDF5 have relied on module scoping. Probably just add these to L39 https://github.com/JuliaIO/HDF5.jl/pull/847/files#diff-3df1923ba1a5ee433e521ee7d3f75d066f7674b297db94ecda3ad89f7fc3e9a2R39

Reply:
I thought it would be useful to export these so that users could do, e.g.
```
f(..., filters=[Filters.Shuffle(), Filters.Deflate(9)])
```